composer.json

```diff
@@ -15,7 +15,7 @@
         "issues": "https://github.com/fe3dback/web-debug-php-core/issues",
         "source": "https://github.com/fe3dback/web-debug-php-core"
     },
-    "minimum-stability": "stable",
+    "minimum-stability": "dev",
```

Comment:
The package is on `master` branch and it's under development mode.
Using `dev` and let `prefer-stable` be true and it can make sure that the dependencies are installed by stable version during `compose install` command executing.